patch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Can we move this file and patch folder to something like `k8s-mojaloop-perf-tuning` or something?

Reply:
Do you mean to move it in a folder or in a new repo? I prefer to have it in this repo, as my plan is later to automate the whole process and use one repo for that task. These patches will be part of the whole script.

Reply:
I meant to move to folder

Reply:
I would suggest to move patch.sh and patch folder into a new folder called something like `k8s-mojaloop-perf-tuning`

Reply:
I moved them to that folder.